__tests__/homepage.test.tsx

```diff
@@ -18,13 +18,77 @@ describe("Home", () => {
     expect(header).toBeInTheDocument();
   });
 
-  it("renders the expected text content", () => {
-    const { getByText } = render(<Home />);
-    expect(getByText("Notification banner")).toBeInTheDocument();
-    expect(getByText("Header")).toBeInTheDocument();
-    expect(getByText("First swim lane")).toBeInTheDocument();
-    expect(getByText("Featured Content")).toBeInTheDocument();
-    expect(getByText("Rest of swim lanes")).toBeInTheDocument();
-    expect(getByText("Explore further links")).toBeInTheDocument();
+  it("renders the Recently Digitized Items row", () => {
+    render(<Home />);
```

Comment:
Btw, for the `swimLanes.test.tsx` file, since it's a component test file and not a _page_ or API test file, it can be colocated in the same directory as the component file. It doesn't have to be in this `__test__` directory.